cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowToken.java

```diff
@@ -71,12 +117,60 @@
   List<NodeValueEntry> getAll(String key);
```

Comment:
If the key is not found, does this return null? If so, mark this as ``@Nullable`` too